src/test/components/commons/Layout.test.js

```diff
@@ -2,13 +2,25 @@ import {render, screen} from '@testing-library/react'
 import {ListWithSelection} from "../../../components/commons/ListWithSelection";
 import {ListElementDetails} from "../../../components/commons/ListElementDetails";
 import userEvent from "@testing-library/user-event";
+import {ThemeProvider} from "@material-ui/styles";
+import {theme} from "../../utils/theme";
 
 describe("Base layout test", () => {
 
+    const renderList = (listItems, extractData, propsHandleSelect) => {
+        render(
+            <ThemeProvider theme={theme}>
+                <ListWithSelection listItems={listItems} propsHandleSelect={propsHandleSelect} extractData={extractData}
+                                   limit={NaN}/>
```

Comment:
Czy tu na pewno powinien być limit NaN?

Reply:
Tak, bo jeśli jest limit NaN, to nie ma limitu XD